src/Lykke.Sdk/AppLifetimeHandler.cs

```diff
@@ -67,6 +69,8 @@ public void HandleStarted()
                     .GetAwaiter()
                     .GetResult();
 
+                if (Debugger.IsAttached)
```

Comment:
This is not the same as #if DEBUG. This property indicates, if debugger is attached - https://msdn.microsoft.com/ru-ru/library/system.diagnostics.debugger.isattached%28v=vs.110%29.aspx?f=255&MSPPError=-2147217396 but we need to see in the logs configuration of the build

Reply:
Local run could be without attached debugger